### PR TITLE
72 countdown timer

### PR DESCRIPTION
**Pull request by kapaha**
Add a countdown timer element that counts down from a certain time and on finish calls game over.

### CODE REVIEW

**Review by tanselbay1 — COMMENTED** [review]
Timer works and looks good but after castle lives finish I see a white page. Before merge this into main. I would recommend you to pull main and test it again. Or we can make new develop branch to test it

**Comment by kapaha** [comment]
Fixed.

File: src/styles/styles.css [hunk 1]
```diff
@@ -125,12 +133,6 @@
     padding: 0.25em 0.5em;
 }
 
-/* ********  GAME-OVER-PAGE  ******** */
-
-#game-over-page {
-    visibility: hidden;
```
Comment (tanselbay1) [inline src/styles/styles.css:131]: Game over page still visible in the beginning of the game

**Comment by tanselbay1** [comment]
looks good to me now

**Pull request merged**